apps/meteor/app/apps/server/bridges/rooms.ts

```diff
@@ -102,6 +102,55 @@ export class AppRoomBridge extends RoomBridge {
 		return this.orch.getConverters()?.get('users').convertById(room.u._id);
 	}
 
+	protected async getMessages(
+		roomId: string,
+		options: {
+			limit: number;
+			skip?: number;
+			sort?: Record<string, 1 | -1>;
+		},
+		appId: string,
+	): Promise<IMessageRaw[]> {
+		this.orch.debugLog(`The App ${appId} is getting the messages of the room: "${roomId}"`);
+
+		let { limit, skip = 0, sort } = options;
+
+		if (!Number.isFinite(limit) || limit < 1) {
+			limit = 100;
+		}
+
+		if (!Number.isFinite(skip) || skip < 0) {
+			skip = 0;
+		}
+
+		if (!sort || typeof sort !== 'object') {
+			sort = { ts: -1 };
+		}
+
+		const messageConverter = this.orch.getConverters()?.get('messages');
+		if (!messageConverter) {
+			throw new Error('Message converter not found');
+		}
+
+		const messageQueryOptions = {
+			limit: Math.min(limit, 100),
+			skip,
+			sort,
+		};
+
+		const query = {
+			rid: roomId,
+			_hidden: { $ne: true },
+			t: { $exists: false },
+		};
+
+		const cursor = Messages.find(query, messageQueryOptions);
```

Comment:
The description talks about `findVisibleByRoomId` but here it's just a normal find 👀

Reply:
Done, thank you!